.github/workflows/ion-java-performance-regression-detector.yml

```diff
@@ -16,10 +16,10 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - name: Set up JDK 1.8
-        uses: actions/setup-java@v1
+      - name: Set up JDK 11
```

Comment:
Is there any point in naming this with the version when that is redundant?

Reply:
You can easily read it from the GHA status page, I suppose. 🤷‍♂️